### DISCUSSION TITLE
No button to manually clear downloaded videos

### DISCUSSION THREAD

**Discussion by cnwtech** [aclap-dev/video-downloadhelper#753] [Help and Q&A]
### Your question

On NEW UI you can see available as well as DOWNLOADED videos in the plug-in pull-down.

There is a SETTINGS OPTION to "Show Clean Button" (to remove discovered media... not helpful when visiting a site to find/download videos) & "Show Clean ALL Button" (to remove all discovered media AND downloaded media HISTORY [ not files ]).   But these options don't help.

I want to be able to manually "Clean downloaded media (history)" from this list.    There is an OPTION to "Automatically Hide Downloaded Media"... but this makes process of managing downloads difficult... I may come back after lengthy download and need to process downloaded video (move to network or USB drive) and want to see recently downloaded media file so I can know to action it... or... manually remove it from history if task completed.

Can you add option on individual media file that has been downloaded to remove it from (history) manually... and/or... add button to "Remove Downloaded Media"?

Another solution would be to change "Show Clean ALL" button to only remove (histpry) of downloaded media... not available media (don't understand this use case).... and you can already manually remove downloaded media using the X at the right of each file.
I don't want to delete all available and all downloads... just the latter.

See screenshots....

![Screenshot 2024-05-28 at 4 37 16 PM](https://github.com/aclap-dev/video-downloadhelper/assets/36424620/93c1d92e-1548-4d7a-97f5-5c00106d2d76)

![Screenshot 2024-05-28 at 4 37 38 PM](https://github.com/aclap-dev/video-downloadhelper/assets/36424620/16120d89-92a1-4acd-b9a4-826164120bb5)

Thanks... G

### Addon & CoApp version and details

version: 9.0.1.13
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"11909f684a654deb"}
platform: aarch64 mac
UA: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:126.0) Gecko/20100101 Firefox/126.0
{
    "valid_license_message_has_been_displayed": true,
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": "/Users/grantl/Downloads/VDH",
    "view_options": {
        "all_tabs": false,
        "low_quality": true,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "use_sidebar": false,
    "open_count_store": 45
}

**Comment by Jackalaq** [comment]
Agreed; this is an issue.  I'd like this too so I'll include myself to hear updates.
